catalog/views.py

```diff
@@ -1,3 +1,17 @@
 from django.shortcuts import render
 
-# Create your views here.
+from catalog.models import Book, Author, BookInstance, Genre
+def index(request):
+  """View function for home page of site."""
+  num_books = Book.objects.all().count()
+  num_instances = BookInstance.objects.all().count()
+  num_instances_available = BookInstance.objects.filter(status__exact='a').count()
```

Comment:
Tìm hiểu về Enum để áp dụng cho code tường minh hơn em nhen
ví dụ: 
```suggestion
  num_instances_available = BookInstance.objects.filter(status__exact=BookStatus.AVAILABLE.value).count()
```